iOS/New/Views/Common/Settings/SettingView.swift

```diff
@@ -762,6 +764,12 @@ extension SettingView {
                 SettingsStore.shared.remove(key: key + Self.passwordKeySuffix)
                 SettingsStore.shared.remove(key: key + Self.cookieKeysKeySuffix)
                 SettingsStore.shared.remove(key: key + Self.cookieValuesKeySuffix)
+                // remove local storage
+                if let sourceId = source?.id, let localStorageKeys = value.localStorageKeys {
+                    for key in localStorageKeys {
+                        SettingsStore.shared.remove(key: "\(sourceId)\(Self.localStoragePrefix)\(key)")
```

Comment:
we should namespace it with the `key` prefix like the other ones, since technically a source could use multiple different web login settings and we wouldn't want the local storage keys to possible conflict.